docs/docs/zh/about/hardware.md

```diff
@@ -45,3 +45,6 @@ DolphinScheduler正常运行提供如下的网络端口配置：
 ## 4. 客户端 Web 浏览器要求
 
 DolphinScheduler 推荐 Chrome 以及使用 Chromium 内核的较新版本浏览器访问前端可视化操作界面
+
+## 5. 时钟同步
```

Comment:
BTW, we should remove the prefix number `5.` here. Best practices of markdown and many markdown lint requests avoid it.

please also remove other leading number prefix in markdown title